roles/opensrp/defaults/main.yml

```diff
@@ -66,3 +66,5 @@ opensrp_redis_port: 6379
 opensrp_redis_max_connections: 25
 opensrp_authencation_cache_ttl: 600
 opensrp_postgres_hba_config_host: "127.0.0.1/32"
+opensrp_postgres_rds_deployment: false
+opensrp_postgres_host: "localhost"
```

Comment:
Would be nice if you gave this variable a more generic name to suggest that Postgres should not be installed (rather than Postgres is installed on RDS).